File: ForgeHub.API/Models/Gym.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ForgeHub.API.Models;

[Table("gyms")]
public class Gym
{
    [Key]
    public long Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public long? OwnerUserId { get; set; }
    public string? LogoUrl { get; set; }
    public string? City { get; set; }
    public bool IsActive { get; set; } = true;
    public DateTime? CreatedAt { get; set; } = DateTime.UtcNow;
    
    [ForeignKey("OwnerUserId")]
    public virtual User? OwnerUser { get; set; }
}

File: ForgeHub.Admin/src/types/gym.ts
export interface Gym {
  id: number;
  name: string;
  ownerUserId?: number | null;
  ownerName?: string;
  logoUrl?: string | null;
  city?: string | null;
  branches?: number;
  members?: number;
  monthlyRevenue?: number;
  status?: string;
  isActive?: boolean;
  createdAt?: string | null;
}
